app/scripts/lib/account-tracker.js

```diff
@@ -312,7 +312,7 @@ export default class AccountTracker {
     try {
       balance = await this._query.getBalance(address);
     } catch (error) {
-      if (error.data.request.method !== 'eth_getBalance') {
+      if (error.data.request?.method !== 'eth_getBalance') {
```

Comment:
If we want to use optional chaning, then lets cover all possible properties that may be undefined 
```suggestion
      if (error?.data?.request?.method !== 'eth_getBalance') {
```

Reply:
Would the optional chain on `error` here be superfluous, as `error` exists only to specify what to do when an exception is thrown from the `try` block on L312?

Reply:
Well error can be anything (a string or object). If it doesn't have a property _data_ it would break the code. Same with the other chained properties. A better approach would be to check if _undefined_ or the likes but since you prefer _optional chaining_ it would be better to follow it all the way.

Reply:
ACK